maps/torch/items/clothing/solgov-hands.dm

```diff
@@ -12,15 +12,19 @@
 
 /obj/item/clothing/gloves/thick/duty/solgov/eng
 	name = "engineering duty gloves"
-	desc = "These black duty gloves are made from durable synthetic materials, and have a lovely orange accent color."
+	desc = "These black duty gloves are made from durable insulated synthetic materials, and have a lovely orange accent color."
 	icon_state = "duty_gloves_eng"
 	item_state = "duty_gloves_eng"
+	siemens_coefficient = 0
+	permeability_coefficient = 0.05
 
 /obj/item/clothing/gloves/thick/duty/solgov/fingerless/eng
 	name = "fingerless engineering duty gloves"
-	desc = "These black duty gloves are made from durable synthetic materials, and have a lovely orange accent color. This pair does not, in fact, protect your fingers."
+	desc = "These black duty gloves are made from durable insulated synthetic materials, and have a lovely orange accent color. This pair does not, in fact, protect your fingers."
 	icon_state = "duty_gloves_eng_fingerless"
 	item_state = "duty_gloves_eng_fingerless"
+	siemens_coefficient = 0.2 // turns out snipping fingers from insuls is a bad idea
+	permeability_coefficient = 0.15
```

Comment:
Snipping fingers from insulated gloves practically makes them useless, I don't see how they would be better than normal gloves or cheap insulated ones like that.